tensorboard/plugins/debugger/tf_debugger_dashboard/tf-debugger-dashboard.html

```diff
@@ -324,7 +338,7 @@ <h2>Continue...</h2>
         overflow: auto;
         padding: 20px 0;
       }
-      #top-right-content-container {
+      #explore-container {
```

Comment:
I actually prefer the original naming. "explore" is vague. It doesn't tell you the location of the pane, and it doesn't tell you what it is (graph / tensor value). WDYT?

Reply:
Hmm, that is true. The reason I changed it is because I ended up with long property names like `topRightContentContainerMinLength`. Do you think we could use a shorter name to describe that part of the UI? Maybe `visual-container`?

Reply:
`topRightContentContainerMinLength` does look too long, how about remove the `ContentContainer` or replace it with `Quadrant`?

Reply:
SG. Renamed to top right quadrant.